bears/java/PMDBear.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class PMDBear(LocalBear, Lint):
+    executable = 'run.sh'
+    arguments = "pmd -R java-basic," + \
+                "java-braces," + \
+                "java-clone," + \
+                "java-codesize," + \
+                "java-controversial," + \
+                "java-design," + \
+                "java-imports," + \
+                "java-naming," + \
+                "java-optimizations," + \
+                "java-strings," + \
+                "java-unnecessary," + \
+                "java-unusedcode" + \
+                " -d {filename}"
+    output_regex = re.compile(
+        r'(?P<file_name>.+):'
+        r'(?P<line>.+):'
+        r'(?P<message>.*)')
+    severity_map = {
+        "error": RESULT_SEVERITY.MAJOR,
+        "warning": RESULT_SEVERITY.NORMAL}
```

Comment:
your regex doesn't contain any ``(?P<severity>.*)``, therefore i think this is totally useless.